website/static/js/fangorn.js

```diff
@@ -430,34 +441,63 @@ function _fangornToggleCheck(item) {
     return false;
 }
 
-function checkConflicts(tb, item, folder, cb) {
-    var messageArray = [];
-    for(var i = 0; i < folder.children.length; i++) {
-        var child = folder.children[i];
-        if (child.data.name === item.data.name && child.id !== item.id) {
-           messageArray.push([
-                m('p', 'An item named "' + child.data.name + '" already exists in this location.'),
-                m('h5.replace-file',
-                    '"Keep Both" will retain both files (and their version histories) in this location.'),
-                m('h5.replace-file',
-                    '"Replace" will overwrite the existing file in this location. ' +
-                    'You will lose previous versions of the overwritten file. ' +
-                    'You will keep previous versions of the moved file.'),
-                m('h5.replace-file', '"Cancel" will cancel the move.')
-            ]);
+function checkConflicts(items, folder){
```

Comment:
Most of the file can't be tested well, but this function could be. Please add some tests.